src/core/evaluator.js

```diff
@@ -1695,6 +1755,21 @@ class PartialEvaluator {
             // e.g. as done in https://github.com/mozilla/pdf.js/pull/6266,
             // but doing so is meaningless without knowing the semantics.
             continue;
+          case OPS.beginMarkedContentProps:
+            if (!isName(args[0])) {
+              warn(`Expected name for beginMarkedContentProps arg0=${args[0]}`);
+              continue;
+            }
+            fn = OPS.beginMarkedContentProps;
```

Comment:
This line looks superfluous, since `fn === OPS.beginMarkedContentProps` must hold in order for this block to have been entered!?